lineman/app/components/group_page/logo_photo_form/logo_photo_form.coffee

```diff
@@ -0,0 +1,14 @@
+angular.module('loomioApp').factory 'LogoPhotoForm', ->
+  scope: {group: '='}
+  templateUrl: 'generated/components/group_page/logo_photo_form/logo_photo_form.html'
+  controller: ($scope, $rootScope, $timeout, group, Records, FlashService) ->
+
+    $scope.selectFile = ->
+      $timeout -> document.querySelector('.group-logo-modal__file-input').click()
+
+    $scope.upload = (files) ->
```

Comment:
Lots of duplication here with the cover_photo_form; a single controller with two templates? They're really similar forms.